components/brave_rewards/core/rewards_callbacks.h

```diff
@@ -23,6 +23,9 @@ using AttestPromotionCallback = mojom::RewardsEngine::AttestPromotionCallback;
 
 using ClaimPromotionCallback = mojom::RewardsEngine::ClaimPromotionCallback;
 
+using BeginExternalWalletLoginCallback =
+    mojom::RewardsEngine::BeginExternalWalletLoginCallback;
+
```

Comment:
Nit: can we move this up under `AttestPromotionCallback` (to keep the alphabetical order)?